src/js/game/LevelMVC/LevelView.js

```diff
@@ -1159,6 +1159,62 @@ module.exports = class LevelView {
     }
   }
 
+  refreshActionPlane(levelData, indices) {
+    var sprite,
+      blockType;
+
+    for (var index in indices) {
+      if (indices[index] >= 0) {
+        let workingIndex = indices[index];
+
+        sprite = this.actionPlaneBlocks[workingIndex];
+        this.actionPlane.remove(sprite);
+
+        let y = Math.floor(workingIndex / this.controller.levelModel.planeHeight);
+        let x = workingIndex - (y * this.controller.levelModel.planeHeight);
```

Comment:
Rather than converting index -> x, y can we pass in an array of positions?  It's easy to go from [x, y] -> index via `coordinatesToIndex`.